external/sqlitejs/sql.js

```diff
@@ -1,6 +1,6 @@
 // This prevents pollution of the global namespace
 var SQL = "Uint8ClampedArray" in window && (function () {
-function f(a){throw a;}var i=void 0,j=!0,k=null,l=!1;function n(){return function(){}}var q,r;r||(r=eval("(function() { try { return Module || {} } catch(e) { return {} } })()"));var aa={},ba;for(ba in r)r.hasOwnProperty(ba)&&(aa[ba]=r[ba]);var ca="object"===typeof process&&"function"===typeof require,da="object"===typeof window,ea="function"===typeof importScripts,fa=!da&&!ca&&!ea;
+function f(a){throw a;}var i=void 0,j=!0,k=null,l=!1;function n(){return function(){}}var q,r;r||(r=eval("(function() { try { return Module || {} } catch(e) { return {} } })()"));var aa={},ba;for(ba in r)r.hasOwnProperty(ba)&&(aa[ba]=r[ba]);var ca=false&&"object"===typeof process&&"function"===typeof require,da="object"===typeof window,ea="function"===typeof importScripts,fa=!da&&!ca&&!ea;
```

Comment:
I disabled this check which was determining if we were "in Node" or not (it's not a very good check, unfortunately).